### ISSUE TITLE
Better logging and in browser log viewer

### ISSUE THREAD

**Issue by powdermilkman** [DEFENDORe/pseudotv#9]
Make logging all around better and make logs viewable in browser app.

**Comment by FangsMcWolf** [comment]
If this is added, and a way to choose what to see (simple activity, full activity, errors, etc), then my request #8 can be ignored.  This would be much better.

**Comment by powdermilkman** [comment]
Ok, I will close your issue.

**Comment by FangsMcWolf** [comment]
Yeah, can view the info remotely and no risk of losing it if the window closes.  Just have to relaunch to access it.  Definitely a better option.
